src/neo/Ledger/StorageItem.cs

```diff
@@ -9,6 +9,9 @@ namespace Neo.Ledger
 {
     public class StorageItem : ICloneable<StorageItem>, ISerializable
     {
+        public const int MaxInteropEntries = 32;
+        public const int MaxIteropEntrySize = 512;
```

Comment:
```suggestion
        public const int MaxInteropEntrySize = 64;
```

Reply:
The problem is that Oracle will use more than this https://github.com/neo-project/neo/blob/ab7ec04925c1e10c9338e3deb8880e93b75ff306/src/neo/SmartContract/Native/Oracle/OracleContract.cs#L23

Reply:
Why don't we reuse VM's MaxStackSize/MaxItemSize here? I've traced back the original 16/34 values to #870 and #1006, back then they only affected Nep5AccountState which was OK given that we knew well the contents of it. Now after #1583 it suddenly has much broader usage and potentially can affect any StorageItem saved from native contracts. So we can of course try to check all current use cases, but there probably will be more and it's easy to forget updating the limit here, so if we're saving stack items, maybe we'd be fine using whatever limits exist for stack items and they're known from VM module.

Reply:
Please take a look, in order to avoid pass the application engine as an argument, i cached the values in the static constructor.